File: lista_1/quermesse.py
N = int(input())
t = 1
while N != 0:
    entrada = list(map(int, input().split()))
    for j in range(N):
        numero = entrada[j] / (j + 1)
        if numero == 1:
            print('Teste', t)
            print(j+1)
            print()
    N = int(input())
    t += 1

File: lista_1/quermesse.rb
convidados = gets.chomp.to_i
teste = 1

while (convidados != 0)
	lista = gets.split.map(&:to_i)
	
	for i in 0...convidados do
		if (lista[i] == i+1)
			puts "Teste #{teste}"
			puts i+1
			puts
    end
  end

	convidados = gets.chomp.to_i
	teste += 1

end
